worker/src/index.ts

```diff
@@ -869,7 +951,8 @@ export default {
       return new Response(
         JSON.stringify({
           "helicone-message":
-            "oh no :( this is embarrassing, Helicone ran into an error proxy-ing your request. Please try again later",
+            "this is embarrassing, Helicone ran into an error with your request. This was the issue: " +
```

Comment:
lol you didnt like my message?

Reply:
Sorryyy, It was misleading because it suggested there was an internal error, whereas I was getting user errors for incorrect formatting and usage. This is not an error with the proxy (forwarding to OpenAI) or a retry issue ("please try again later"). Does that make sense?